style.css

```diff
@@ -0,0 +1,79 @@
+.header-container {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 718px;
+  margin-left: 20px;
+  background-image: url('/images/header-shapes mobile@2x.png');
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
```

Comment:
Kindly make sure that your design matches the figma template as much as possible🙏. This included fonts, spacing, colors, and padding. 

Your design:
<img width="195" alt="image" src="https://user-images.githubusercontent.com/37116322/219207060-236fa1b8-e8b1-4d2c-b994-69731085f726.png">

Figma design:
<img width="195" alt="image" src="https://user-images.githubusercontent.com/37116322/219207665-6d64e1b8-5b54-4d4b-8ebb-d1ef6426714f.png">